lib/Map/DisplayVariablesConcept.js

```diff
@@ -75,12 +75,12 @@ inherit(Concept, DisplayVariablesConcept);
 
 defineProperties(DisplayVariablesConcept.prototype, {
     /**
-     * Gets a value indicating whether this item has child items.
+     * Gets a value indicating whether this item has visible child items.
      * @type {Boolean}
      */
     hasChildren : {
         get : function() {
-            return this.items.length > 0;
+            return this.items.filter(function(concept) {return concept.isVisible;}).length > 0;
```

Comment:
Minor, but this can be slightly more efficiently written as `this.items.some(function(concept) { return concept.isVisible; });`